src/ripple/app/ledger/Ledger.h

```diff
@@ -258,57 +242,9 @@ class Ledger
     uint256 const&
     getHash();
 
-    uint256 const& getParentHash () const
-    {
-        return mParentHash;
-    }
-
-    uint256 const& getTransHash () const
-    {
-        return mTransHash;
-    }
-
-    uint256 const& getAccountHash () const
-    {
-        return mAccountHash;
-    }
-
-    std::uint64_t getTotalCoins () const
-    {
-        return mTotCoins;
-    }
-
-    void setTotalCoins (std::uint64_t totCoins)
+    void setTotalCoins (std::uint64_t totDrops)
```

Comment:
it'd be nice to do away with this function, we might not need to set total coins any more. We should check callers. Perhaps best in a separate PR